src/estimation/include/iDynTree/Estimation/ContactStateMachine.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Silvio Traversaro, Prashanth Ramadoss
+ * email: silvio.traversaro@iit.it, prashanth.ramadoss@iit.it
+ *
+ * Permission is granted to copy, distribute, and/or modify this program
+ * under the terms of the GNU General Public License, version 2 or any
+ * later version published by the Free Software Foundation.
+ *
+ * A copy of the license can be found at
+ * http://www.robotcub.org/icub/license/gpl.txt
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details
+ */
+
+#ifndef CONTACTSTATEMACHINE_H
```

Comment:
Can you add `IDYNTREE_` to the guard macro?

Reply:
Note: this applies also to the other classes.

Reply:
Fixed it.